public/js/components/AtomActions/NotificationList.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component, PropTypes } from 'react';
+import {atomPropType} from '../../constants/atomPropType';
+
+class NotificationList extends Component {
+
+  static propTypes = {
+    actions: PropTypes.shape({
+      createNotificationList: PropTypes.func.isRequired,
+      deleteNotificationList: PropTypes.func.isRequired
+    }).isRequired,
+    atom: atomPropType
+  }
+
+  constructor(props) {
+    super(props);
+  }
+
+  getListData(atom) {
+    return (atom.data && atom.data.storyquestions && atom.data.storyquestions.notifications);
+  }
+
+  createNotificationList() {
+    this.props.actions.createNotificationList(this.props.atom);
+  }
+
+  deleteNotificationList() {
+    this.props.actions.deleteNotificationList(this.props.atom);
+  }
+
+  render() {
+    const listData = this.getListData(this.props.atom);
```

Comment:
this belongs in the contructor

Reply:
it needs to check the status of the list each time the atom data updates - if it's in the constructor this won't happen?

Reply:
hmm, then it is a piece of state shared between parent-child components. that's a common pitfall in React and there are many ways to solve it, one of them being via properties; perhaps the editorial team prefers one style over another, I'll leave it up to them 👍